### PR TITLE
ROC-5418: Helm config changes

### PR DESCRIPTION
**Pull request by markry-kainos**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-5418

### Change description

Helm changes required from RPE and some variable cleanup.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: charts/cmc-citizen-frontend/values.aat.template.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+nodejs:
+  environment:
```
Comment (adusumillipraveen) [inline charts/cmc-citizen-frontend/values.aat.template.yaml]: You will still need image and ingressHost templated fields in aat file.

File: charts/cmc-citizen-frontend/values.aat.template.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
+    IDAM_AUTHENTICATION_WEB_URL: https://idam-web-public.aat.platform.hmcts.net
+    IDAM_S2S_AUTH: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
```
Comment (adusumillipraveen) [inline charts/cmc-citizen-frontend/values.aat.template.yaml]: You need them in .preview.template.yaml as well We have moved them to values.yaml to be sensible defaults.?

Comment (adusumillipraveen) [reply charts/cmc-citizen-frontend/values.aat.template.yaml]: Same with other URLs as well if they are applicable?

File: charts/cmc-citizen-frontend/values.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+nodejs:
+  image: hmcts.azurecr.io/hmcts/cmc-claim-store:latest
```
Comment (KiranVarma02) [inline charts/cmc-citizen-frontend/values.yaml]: is this right image name? nodejs and claim-store!!!

Comment (markry-kainos) [reply charts/cmc-citizen-frontend/values.yaml]: no its not!!!! 🙈 will fix

**Pull request merged**